<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WarClientClass</class>
 <widget class="QMainWindow" name="WarClientClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1640</width>
    <height>1200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1640</width>
    <height>1200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1640</width>
    <height>1200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>WarClient</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="AddModelBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>200</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>模型添加</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="ModelLogEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>400</y>
      <width>500</width>
      <height>800</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="MsgLogEdit">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>400</y>
      <width>500</width>
      <height>800</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>350</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>模型日志信息</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>350</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>系统日志信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StartEngineBtn">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>0</y>
      <width>200</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>运行引擎</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>150</y>
      <width>200</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>   运行状态:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EngineStatusEdit">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>150</y>
      <width>200</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>250</y>
      <width>200</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>   系统时间:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SystemTimeEdit">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>250</y>
      <width>300</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>10</y>
      <width>200</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>我方单位总览</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ServialModelBtn">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>10</y>
      <width>200</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>现存装备</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DestoryedModelBtn">
    <property name="geometry">
     <rect>
      <x>1400</x>
      <y>10</y>
      <width>200</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>毁伤装备</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ScenarioEditBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>200</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>想定编辑</string>
    </property>
   </widget>
   <widget class="QPushButton" name="OpenScenBtn">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>200</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>打开想定</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="PosBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>270</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>坐标</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="attackBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>300</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>攻击</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="destoryBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>330</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>毁伤</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cmdBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>360</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>指令</string>
    </property>
   </widget>
   <widget class="QComboBox" name="CampBox">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>85</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>85</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>阵营:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Situation2DBtn">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>100</y>
      <width>200</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>二维态势</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>85</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>步长:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="LengthBox">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>85</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>500</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>300</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>100</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>50</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>服务器IP:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ServerIPEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>180</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ConnectBtn">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>180</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>连接</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>连接状态:</string>
    </property>
   </widget>
   <widget class="QLabel" name="ConnectStatusLab">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>220</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>未连接</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1640</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="WarClient.qrc"/>
 </resources>
 <connections/>
</ui>
